modules/compliance-anatomy.adoc

```diff
@@ -209,7 +228,8 @@ $ oc annotate compliancescans/<scan_name> compliance.openshift.io/rescan=
 
 After the scan reaches the Done phase, nothing else happens on its own unless the remediations are set to be applied automatically with `autoApplyRemediations: true`. The {product-title} administrator would now review the remediations and apply them as needed. If the remediations are set to be applied automatically, the `ComplianceSuite` controller takes over in the Done phase, pauses the machine config pool to which the scan maps to and applies all the remediations in one go. If a remediation is applied, the `ComplianceRemediation` controller takes over.
 
-== `ComplianceRemediation` lifecycle and debugging
+[id="compliance-remediation-lifecycle-debugging_{context}"]
+== Compliance Remediation lifecycle and debugging
```

Comment:
Similar feedback: s/Compliance Remediation lifecycle and debugging/Compliance remediation lifecycle and debugging/ or /ComplianceRemediation controller lifecycle and debugging/